api/internal/handler/route/route_test.go

```diff
@@ -754,6 +754,20 @@ func TestRoute(t *testing.T) {
 	_, err = handler.Update(ctx)
 	assert.Nil(t, err)
 
+	//sleep
+	time.Sleep(time.Duration(100) * time.Millisecond)
+	
+	// check ID discrepancy on Update
+	errRoute := &UpdateInput{}
+	errRoute.ID = "2"
+	err = json.Unmarshal([]byte(reqBody), errRoute)
+	assert.Nil(t, err)
+	ctx.SetInput(errRoute)
+	ret, err = handler.Update(ctx)
+	assert.NotNil(t, err)
+	assert.EqualError(t, err, "ID on path (2) doesn't match ID on body (1)")
+	assert.Equal(t, http.StatusBadRequest, ret.(*data.SpecCodeResponse).StatusCode)
+
```

Comment:
For historical reasons, id supports int and stringing.
so we need test case for int ID in body.

Reply:
I think this test should support